libbpf-tools/klockstat.h

```diff
@@ -0,0 +1,19 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
+#pragma once
```

Comment:
Consider replace this `#pragma once` with `#ifndef` guard.

Reply:
Done